onnxruntime/core/providers/cuda/tensor/resize.cc

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "resize.h"
+
+namespace onnxruntime {
+namespace cuda {
+#define REGISTER_KERNEL_TYPED(T)                                  \
+  ONNX_OPERATOR_TYPED_KERNEL_EX(                                  \
+      Resize,                                                     \
```

Comment:
>Resize [](start = 6, length = 6)

do we have UT for Resize?

Reply:
yes.
https://github.com/Microsoft/onnxruntime/blob/master/onnxruntime/test/providers/cpu/tensor/resize_op_test.cc